pkg/rtc/mediatrack.go

```diff
@@ -38,6 +38,7 @@ type MediaTrackParams struct {
 	SdpCid              string
 	ParticipantID       livekit.ParticipantID
 	ParticipantIdentity livekit.ParticipantIdentity
+	ParticipantVersion  uint32
```

Comment:
Using `ParticipantVersion` as a defensive guard. If add/remove get out of order (actual sequence wants to be remote remove -> local add, but if that gets flipped for some reason, subscribed track could be removed incorrectly). So, using version to ensure that add/remove subscribed track does not happen on a stale version. Ideally add should not need that check, but using it for consistency.